script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5.txt

```diff
@@ -170,19 +170,19 @@ and the thing that should have fallen, rises gently instead.
 Like a sailing ship catching the wind, she gambols through the night sky.
 <Page48>
 There's nothing to be surprised about.
-That incarnation is a projection of a child from another planet.
-All the fluctuations that occur on the surface belong to her.
+That figure is an avatar of the planet itself.//その化身は 惑星の 嬰児にして写身。
+All the fluctuations that occur on its surface belong to her.
 <Page49>
-The blessings of nature that give way to life,
-the wrath of nature that takes it away,
-the naturally endowed rights of that body.
+The blessings of nature that nurture life,
+the wrath of nature that takes it away.
+Of course such a body would be posessed of these rights.//その体に備わった当然の権利である。
 <Page50>
 "Okay―――"
-The beautiful flower turns itself to look at the moon.
+The beautiful flower twists around, as if to look up at the moon.
 Her consciousness turns from the sky to the ground.
 The unraveled chains and the raised anchors strike the surface of the Earth.
 Below her is a tall, stone building scattering fire.
-"―――Let's do this!"
+"―――Here we go!"
 <Page51>
-The blooming flower in the night sky turns into a falling star.
-The flower of the moon transforms itself into a tempest, blowing away the burning blaze.
+The blossom that bloomed in the night sky turns into a falling star.
```

Comment:
I think remove 'that bloomed' and just keep blossom, what is there now just sounds really unpolished.
Blossom as a verb has the same meaning as bloom so I think enough of the connotations will be conveyed despite us using it as a noun.